formatcore/base.go

```diff
@@ -1,4 +1,4 @@
-package format
+package formatcore
```

Comment:
Can you move the `formatcore` package inside of the `format` package? It may seem a bit odd with the repeated name, but it's a reasonably common pattern to get a good import name with proper logical grouping (see: [`io/ioutil`](https://golang.org/pkg/io/ioutil/) and [`zap/zapcore`](https://pkg.go.dev/go.uber.org/zap@v1.17.0/zapcore)). So formatcore/base.go would become format/formatcore/base.go

Reply:
Oh right, looks better this way. This should be good now